pkg/limits/store.go

```diff
@@ -7,6 +7,9 @@ import (
 	"github.com/grafana/loki/v3/pkg/limits/proto"
 )
 
+// The number of stripe locks.
+const numStripes = 64
```

Comment:
We might decide to make this configurable in the future, but for now I just want to separate it out as its own const.